browser/extensions/manifest_v2/brave_hosted_extensions.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/extensions/manifest_v2/brave_hosted_extensions.h"
+
+namespace extensions_mv2 {
+
+namespace {
+consteval bool CheckExtensionMaps() {
```

Comment:
nit: Please add a comment giving a higher level explanation of what this function does, because we are looping in their over two sets of static data, so that can be a bit vague.